app/models/snp.rb

```diff
@@ -43,6 +46,15 @@ def self.update_frequencies
     end
   end
 
+  def self.update_phenotypes
+    max_age = 31.days.ago
+
+    Snp.select([:id, :phenotype_updated]).where(phenotype_updated < max_age)
+       .find_each do |snp|
+      Sidekiq::Client.enqueue(LinkSnpPhenotype, snp.id)
```

Comment:
Ah. Okay. That's how I was running the test job in console. Just curious about the preference? Both are performed asynchronously anyway, are they not? @tsujigiri